bundles/org.openhab.binding.ecoflow/src/main/java/org/openhab/binding/ecoflow/internal/handler/PowerStreamHandler.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.ecoflow.internal.handler;
+
+import static org.openhab.binding.ecoflow.internal.EcoflowBindingConstants.PowerStreamChannels.*;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.UnDefType;
+
+import com.google.gson.JsonObject;
+
+/**
+ * @author Danny Baumann - Initial contribution
+ */
+@NonNullByDefault
+public class PowerStreamHandler extends AbstractEcoflowHandler {
+    private static final ValueConverter INVERTER_STATUS_CONVERTER = value -> {
+        return switch (value.getAsInt()) {
+            case 1 -> new StringType("idle");
+            case 2 -> new StringType("starting");
+            case 6 -> new StringType("synchronized");
+            case 11 -> new StringType("disconnected");
+            default -> UnDefType.NULL;
+        };
+    };
+    private static final ValueConverter SUPPLY_PRIORITY_CONVERTER = //
+            value -> new StringType(value.getAsInt() > 0 ? "prioIsStorage" : "prioIsSupply");
+    private static final ValueConverter INV_SWITCH_CONVERTER = value -> value.getAsInt() != 0 ? OnOffType.OFF
+            : OnOffType.ON;
+    private static final ValueConverter PERCENT_DECIMAL_CONVERTER = value -> new DecimalType(value.getAsNumber());
+    private static final ValueConverter PERCENT_DIMMER_CONVERTER = value -> new PercentType(value.getAsInt());
+
+    private static final List<ChannelMapping> MAPPINGS = List.of(
+            new ChannelMapping("20_1", "invStatue", CHANNEL_ID_INV_STATUS, INVERTER_STATUS_CONVERTER),
```

Comment:
invStatue or invStatus ?

Reply:
`invStatue` is (sadly) correct, see [their docs](https://developer-eu.ecoflow.com/us/document/powerStreamMicroInverter)